test/Microsoft.Framework.ApplicationHost.FunctionalTests/AppHostTests.cs

```diff
@@ -107,7 +107,7 @@ public void AppHostShowsErrorWhenGivenSubcommandWasNotFoundInProjectJson(string
         {
             var runtimeHomeDir = _fixture.GetRuntimeHomeDir(flavor, os, architecture);
             var projectStructure = @"{
-  'project.json': '{ }'
```

Comment:
Yes. I remove the support. Single quote is not supported according to JSON spec.